assets/js/script.js

```diff
@@ -11,15 +76,13 @@ const getToken = () => {
         data: `grant_type=client_credentials&client_id=${ID}&client_secret=${SECRET}`,
         // Upon Success, stores token with a key of 'token'.
         success: function(data) {
-            getAnimals(data);
+            localStorage.setItem('token', data.access_token);
         },
         dataType: "json"
-      })
+    })
 };
-
-// // Uses token to access array of animals, will need to add location query eventually
+// Uses token to access array of animals, will need to add location query eventually
 const getAnimals = (data) => {
```

Comment:
In `getAnimals()` delete `(data)` parameter, and change `data.access_token` to `TOKEN`